### ISSUE TITLE
Multiple accounts with same email address

### ISSUE THREAD

**Issue by abokth** [fabric8io/fabric8-online-docs#199]
I signed up for openshift.io and had multiple accounts at developer.redhat.com with the same email address. Due to this I got stuck at a login dialog which wouldn't accept the password for the pre-filled-in username. I was able to clear this up by switching the email address on one account, but it would have been simpler to do this before signing up, to avoid confusion. This might be something to document in the getting started guide, unless the experience can be improved.

**Comment by rkratky** [comment]
@alexeykazakov Is this a bug in auth? It sounds somewhat similar to https://github.com/openshiftio/openshift.io/issues/2332

**Comment by alexeykazakov** [comment]
Yes, this is related to https://github.com/openshiftio/openshift.io/issues/2332 but a bit different issue I think.. Looks like developers.redhat.com couldn't login the user at all. I will ping Vlastimil (RHD) to investigate.

**Comment by velias** [comment]
@abokth we changed developers.redhat.com IdP (used by openshift.io also) recently to fully support existing multiple accounts with same email address. We tried to implement rules which provide good backward compatibility for users who used to login with email. It is really strange if you was not able to login for some account at all. Could you please provide me more info so I can investigate what exactly happened? Info I need is username entered into the login form. Also list of your usernames and emails used for them at the time of login (as you changed some of them and it is not clear which one). You can send this info to my email from Github user profile not to expose it publicly. Thanks.

**Comment by rkratky** [comment]
@velias @alexeykazakov Does this need to be tracked here, or has it been resolved?

**Comment by alexeykazakov** [comment]
I don't know if this issue was resolved for @abokth or not. But overall https://github.com/openshiftio/openshift.io/issues/2332 issue is still there. We improved the login flow for accounts with the same email but it's not fixed completely.
Maybe its worth adding a workaround to some sort of FAQ for OSIO. Users can try to use username instead of email while logging into OSIO if they face https://github.com/openshiftio/openshift.io/issues/2332.
